src/components/Collapsible/Collapsible.tsx

```diff
@@ -66,20 +64,24 @@ const Collapsible: React.FC<CollapsibleProps> = ({
 
   return (
     <Container className={className} withBackground={isOpen}>
-      <Header alignItems="center" onClick={handleHeaderClick}>
-        <StyledIcon
-          isRotated={isOpen}
-          name={SSCIconNames.chevronRight}
-          type={IconTypes.ssc}
-        />
-        <div>
-          <Text size={TextSizes.md}>{title}</Text>
-          <Subject>{subject}</Subject>
-        </div>
+      <Header paddingSize="lg" paddingType="squish" onClick={handleHeaderClick}>
+        <Inline align="center" gap="lg">
```

Comment:
{SpaceSizes.lg}

Reply:
You can remove `margin-right` property in `StyledIcon`